README.adoc

```diff
@@ -339,7 +339,17 @@ Avoid trailing whitespace.
 
 === One File Per Namespace [[one-file-per-namespace]]
 
-Use one file per namespace.
+Use exactly one file per namespace.
```

Comment:
I guess there are two aspects to this - having a namespace over multiple files and having several namespaces in the same file (e.g. using `in-ns`).

Reply:
Gotcha. I think I understand the direction things should change in